src/qmfunctions/density_utils.cpp

```diff
@@ -51,12 +51,27 @@ namespace density {
 Density compute(double prec, Orbital phi, DensityType spin);
 void compute_local_X(double prec, Density &rho, OrbitalVector &Phi, OrbitalVector &X, DensityType spin);
 void compute_local_XY(double prec, Density &rho, OrbitalVector &Phi, OrbitalVector &X, OrbitalVector &Y, DensityType spin);
-double compute_occupation(Orbital &phi, DensityType dens_spin);
+double compute_occupation(const Orbital &phi, DensityType dens_spin);
 } // namespace density
 
+double density::compute_occupation(const Orbital &phi, DensityType dens_spin) {
+    double occ_a(0.0), occ_b(0.0), occ_p(0.0);
+    if (phi.spin() == SPIN::Alpha) occ_a = (double)phi.occ();
```

Comment:
it's c++ so verbosity is king!
```suggestion
    if (phi.spin() == SPIN::Alpha) occ_a = static_cast<double>(phi.occ());
```